docs/mg7-paper/paper.md

```diff
@@ -121,11 +121,25 @@ Bio4j [@pareja2015bio4j] is a data platform integrating data from different reso
+The starting point for our 16S reference database is RNA Central [@rnacentral2014rnacentral], [version 5][rnacentral v5]. RNACentral was chosen being the most up to date, comprehensive RNA sequence repository, including among others all RNA data from Silva, GreenGenes, RDP, ENA, and RefSeq<!-- TODO cite them -->. First we take those sequences which
 
-Certainly, if metagenomics results are easily integrated with the theoretical and experimental knowledge of each specific area, the impact of metagenomics will be higher than if it progresses as a disconnected research branch. Considering that metagenomics data interoperability, which is especially critical in clinical environments, requires a stable taxonomy to be used as reference, we decided to rely on the most widely used taxonomy: the NCBI taxonomy. In addition, the biggest global sequence database GenBank follows this taxonomy to register the origin of all their submitted sequences.
-Our 16S database building strategy allows the substitution of the 16S database by any other subset of **nt**, even by the complete **nt** database if it would be needed, for example, for analyzing shotgun metagenomics data. This possibility of changing the reference database provides flexibility to the system enabling it for easy updating and project-driven personalization.
+1. are annotated as being of `rRNA` type^[We are aware of the existence of a gene annotation corresponding to 16S in RNACentra, that we are **not using** due to a significant amount of 16S sequences lacking it]
+2. their length is at least `1300`^[Note that 16S sequences are sometimes part of an entry corresponding to whole small subunits; that's why we do not set a maximum length threshold]
+3. have at least one taxonomic assignment to a descendant of *Bacteria* or *Archaea*
+4. their lineage does not contain a set of taxa deemed uninformative^[for example: "unclassified Bacteria (miscellaneous)" or "unclassified". Assigning a read to an "unclassified" taxon defeats the first and foremost goal of taxonomic profiling: *classification*.]
```

Comment:
Isn't it too many footnotes for one paragraph? Except the first one, why not to just include this text here?